doc/nrf/ug_bootloader.rst

```diff
@@ -7,212 +7,198 @@ Secure bootloader chain
 
-1. Append the following configuration to the :file:`ncs/bootloader/mcuboot/boot/zephyr/prj.conf` file::
+MCUboot partitions
+------------------
 
-      CONFIG_NORDIC_QSPI_NOR=y
-      CONFIG_NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
-      CONFIG_NORDIC_QSPI_NOR_FLASH_ALLOW_STACK_USAGE_FOR_DATA_IN_FLASH=y
-      CONFIG_MULTITHREADING=y
-      CONFIG_BOOT_MAX_IMG_SECTORS=256
-      CONFIG_PM_EXTERNAL_FLASH=y
-      CONFIG_PM_EXTERNAL_FLASH_DEV_NAME="MX25R64"
-      CONFIG_PM_EXTERNAL_FLASH_SIZE=0xf4000
-      CONFIG_PM_EXTERNAL_FLASH_BASE=0
+MCUboot requires two image slots:
 
-   These options enable the QSPI NOR flash driver, multi-threading (which is required by the flash driver), and the external flash of the nRF52840 DK.
-#. Update the :file:`ncs/bootloader/mcuboot/boot/zephyr/pm.yml` file (as described above)::
+* The *primary slot*, containing the application to be booted.
+* The *secondary slot*, where a new application can be stored before it is activated.
 
-      mcuboot_secondary:
-          region: external_flash
-          size: CONFIG_PM_EXTERNAL_FLASH_SIZE
-
-#. Add the following configuration to the :file:`prj.conf` file in your application directory::
-
-      CONFIG_NORDIC_QSPI_NOR=y
-      CONFIG_NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
-      CONFIG_NORDIC_QSPI_NOR_FLASH_ALLOW_STACK_USAGE_FOR_DATA_IN_FLASH=y
-      CONFIG_PM_EXTERNAL_FLASH=y
-      CONFIG_PM_EXTERNAL_FLASH_DEV_NAME="MX25R64"
-      CONFIG_PM_EXTERNAL_FLASH_SIZE=0xf4000
-      CONFIG_PM_EXTERNAL_FLASH_BASE=0
+See the *Image Slots* section in the :doc:`MCUboot documentation <mcuboot:design>` for more information.
 
-   These options enable the QSPI NOR flash driver and the external flash of the nRF52840 DK.
-   Multi-threading is enabled by default, so you do not need to enable it again.
+The |NCS| variant of MCUboot uses the :ref:`partition_manager` to configure the flash partitions for these image slots.
+In the default configuration, defined in :file:`bootloader/mcuboot/boot/zephyr/pm.yml`, the partition manager dynamically sets up the partitions as required for MCUboot.
+For example, the partition layout for :file:`zephyr/samples/hello_world` using MCUboot on the ``nrf52840dk_nrf52840`` development kit would look like the following:
+
+.. NOTE: The following ASCII blocks render strangely if the first character before the parenthesis is not a whitespace
+
+.. code-block:: console
+
+    (0x100000 - 1024.0kB):
+   +-----------------------------------------+
+   | 0x0: mcuboot (0xc000)                   |
+   +---0xc000: mcuboot_primary (0x7a000)-----+
+   | 0xc000: mcuboot_pad (0x200)             |
+   +---0xc200: mcuboot_primary_app (0x79e00)-+
+   | 0xc200: app (0x79e00)                   |
+   | 0x86000: mcuboot_secondary (0x7a000)    |
+   +-----------------------------------------+
+
+If you want to control where in memory the flash partitions are placed, you can define :ref:`static partitions <ug_pm_static>` for your application.
```

Comment:
You should change the kconfig to get the sizes as you want them. 

You only use static partitions after you have shipped a product so that you can no longer change the partition layout.

Reply:
When [relocating this section to partition manager docs](https://github.com/nrfconnect/sdk-nrf/pull/3607#discussion_r607163773), I removed this sentence. I wasn't sure how best to word it with extensive docs about this already being present.